crates/ssr/src/matching.rs

```diff
@@ -283,7 +283,12 @@ impl<'db, 'sema> Matcher<'db, 'sema> {
             if p.kind() == SyntaxKind::COMMA && is_closing_token(code.kind()) {
                 pattern.next();
             }
+        } else if code.kind() == SyntaxKind::SEMICOLON {
```

Comment:
To make sure this only applies to let statements, you can add:
`&& code.parent().kind() == SyntaxKind::LET_STMT`

Reply:
`.parent()` is a nice one, definitely simplifies the implementation around making it optional for let statements only :+1: